docs/migration/dfx-0.13.1-migration-guide.md

```diff
@@ -0,0 +1,143 @@
+# 0.13.1 Migration Guide - Environment Variables
+
+This is not an immediate breaking change, but there will be a standardization around environment variables. Backwards compatibility will be supported until 0.15.0, but we recommend updating your projects to use the new environment variables.
+
+## Context
+
+Previously, `webpack.config.js` included an `initCanisterEnv` that parsed canister*ids.json files and mapped them to environment variables. The pattern used there was `<CANISTER_NAME_UPPERCASE>_CANISTER_ID`. This was not consistent with the pattern used in the `dfx` internals, which used `CANISTER_ID*<canister*name_case_agnostic>`. This was confusing for users, and since uppercase environment variables is a best practice, we have decided to standardize on the `CANISTER_ID*<CANISTER_NAME_UPPERCASE>` pattern across the board.
```

Comment:
Maybe I am not understanding something about the use of `*` here.

```suggestion
Previously, `webpack.config.js` included an `initCanisterEnv` that parsed canister*ids.json files and mapped them to environment variables. The pattern used there was `<CANISTER_NAME_UPPERCASE>_CANISTER_ID`. This was not consistent with the pattern used in the `dfx` internals, which used `CANISTER_ID_<canister
_name_case_agnostic>`. This was confusing for users, and since uppercase environment variables is a best practice, we have decided to standardize on the `CANISTER_ID_<CANISTER_NAME_UPPERCASE>` pattern across the board.
```